src/gpsread.cpp

```diff
@@ -76,9 +76,11 @@ void gps_storelocation(gpsStatus_t *gps_store) {
       (gps.location.age() < 1500)) {
     gps_store->latitude = (int32_t)(gps.location.lat() * 1e6);
     gps_store->longitude = (int32_t)(gps.location.lng() * 1e6);
+#if !(PAYLOAD_OPENSENSEBOX)
     gps_store->satellites = (uint8_t)gps.satellites.value();
```

Comment:
We need satellites value for display, also if we do not want opensensebox payload.
Better leave the struct untouched by the compile flag.